### PR TITLE
Reflect shipped 2.0 in roadmap

### PR DESCRIPTION
**Pull request by karelz**
* Add .NET Core 2.0 ship date
* Update .NET Core 2.1 ship date estimate (can change in future)
* Delete irrelevant .NET Core 2.0 information
* Update next focus

Fixes #882

### CODE REVIEW

**Comment by karelz** [comment]
cc @richlander @terrajobst

**Comment by terrajobst** [comment]
Rich is working on an a new roadmap. Stay tuned.

**Comment by karelz** [comment]
@richlander is there ETA? Would it be valuable to update the dates at least in the mean time?

File: roadmap.md [hunk 1]
```diff
@@ -90,11 +40,14 @@ to be an exhaustive list.
...
 
 # Supported OS Versions
 
+## .NET Core 2.1 - Supported OS Versions
+
+Proposal: TBD
+
 ## .NET Core 2.0 - Supported OS Versions
 
 Proposal:
```
Comment (svick) [inline roadmap.md]: Shouldn't the supported OS versions for .Net Core 2.0 be more than just a proposal now?

Comment (karelz) [inline roadmap.md]: I intentionally left 2.0 as it is now - we need to move it into 2.0 release-notes. And I've heard @richlander plans major surgery on the table.

**Comment by richlander** [comment]
I have a larger roadmap re-write currently in review. I'll send it to you.

**Pull request merged**